scripts/provision.sh

```diff
@@ -329,6 +361,10 @@ options:
 
   --install-frontend    Provision a Frontend/API only.
 
+  --install-postgresql  Provision the datastore only.
+
+  --install-minio       Provision the minio server only.
+
   --generate-bootstrap   Generate a bootstrap to be used for scaling our API Frontends
```

Comment:
This isn't related to this change but while you are here, could you delete this line? We removed the `--generate-bootstrap` functionality but did not remove this doc note.